netci.groovy

```diff
@@ -2554,6 +2489,30 @@ def static calculateBuildCommands(def newJob, def scenario, def branch, def isPR
 
                         Utilities.addArchival(newJob, "coreroot.${lowerConfiguration}.zip,testnativebin.${lowerConfiguration}.zip", "")
                     }
+                    else {
+                        assert architecture == 'arm64'
+
+                        // Then, using the same docker image, build the tests and generate the CORE_ROOT layout.
+                        // Linux/arm64 does not use Windows-built tests.
+
+                        def testBuildOpts = ""
+                        if (priority == '1') {
+                            testBuildOpts = "priority1"
+                        }
+
+                        buildCommands += "${dockerCmd}\${WORKSPACE}/build-test.sh ${lowerConfiguration} ${architecture} cross ${testBuildOpts}"
```

Comment:
I think cross is not required?

Reply:
hmmm... I think it is. Egor was working on removing it (and `crosscomponent`) but hasn't finished, I think.

Reply:
It's still required - `cross crosscomponent` will be removed after cross-bitness crossgen becomes default